transformer_engine/common/include/transformer_engine/fused_attn.h

```diff
@@ -0,0 +1,216 @@
+/*************************************************************************
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See LICENSE for license information.
+ ************************************************************************/
+
+#ifndef TRANSFORMER_ENGINE_FUSED_ATTN_FP8_H_
+#define TRANSFORMER_ENGINE_FUSED_ATTN_FP8_H_
+
+#include "transformer_engine.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum NVTE_QKV_Layout {
```

Comment:
I would say this is the most confusing thing that people will encounter working with this API, so we should probably spend more space on explaining the different layouts...

Reply:
Added more layout description.